### DISCUSSION TITLE
Emoji names internationalization

### DISCUSSION THREAD

**Discussion by Finaldrace** [missive/emoji-mart#896] [Ideas]
Would be nice to have the names of the emojis with a customizable language.
(and later maybe the ability to search in the language)

❤️ Red Heart 
❤️ Rotes Herz

(this now depricated repository had some attempts https://github.com/joeattardi/picmo/issues/75)

Source for Localization -> Unicode Common Locale Data Repository (CLDR) 
https://unicode.org/Public/cldr/44/
 cldr-common*.zip ->  annotations

**Comment by karnllp** [comment]
grg

**Comment by wildsonc** [comment]
I used [emojibase](https://emojibase.dev/) to do the translation, it was very simple using cdn.

![CleanShot 2024-04-11 at 16 08 31](https://github.com/missive/emoji-mart/assets/72408418/a01935e2-0b4b-47ab-80ac-3bc1e89dd6d4)

German:

```javascript
<Picker
  data={async () => {
    const response = await fetch(
      "https://cdn.jsdelivr.net/gh/wildsonc/emoji-mart-i18n@latest/locales/de/data.json",
    );

    return response.json();
  }}
  locale="de"
/>
```

[Source code](https://github.com/wildsonc/emoji-mart-i18n)

**Reply by rugk** [reply]
This looks cool but should imho be included in the lib!

Backlink: https://github.com/rugk/awesome-emoji-picker/issues/183
